docs/staking-modules/csm/penalties.md

```diff
@@ -0,0 +1,62 @@
+
+The third penalty type is calculated using the validator withdrawal balance (actual reporting is described in the section below). This penalty is applied immediately within the reporting transaction. If the initial slashing penalty is applied (first penalty type), it will be accounted for to avoid double penalization.
+
+## Immediate and delayed
+The following penalization schemes are introduced:
+1. **Immediate penalization** (for penalties that are unambiguous and can be assessed via trustless proofs);
+2. **Delayed penalty with challenge period** (for cases where false positives may occur or investigation might be needed);
+
+The challenge period for delayed penalties is implemented by separating two roles involved in the application of the penalty. 
+
+The first role is the "reporter". Members of this role can initially report a fact that should result in a penalty. Bond funds will be locked but not burned or confiscated at this stage. "Reporters" can also revoke the initial report in case of the challenge resolution in favor of the Node Operator.
+
+The second role is called "settler". Members of this role can finalize (settle) previously reported penalties.
+
+Separating these two roles ensures that a penalty can only be applied when two independent actors agree. 
+
+## Mechanics
+There are two mechanics related to Node Operator bond penalization.
+
+The first one is burning stETH shares using the [Burner](../../contracts/burner). Once confiscated shares are burnt, the total amount of stETH shares decreases. Hence, `shareRate` increases, effectively distributing all burned stETH value between other stETH holders.
+
+The second mechanic is transferring confiscated stETH to the [Lido DAO Treasury](https://etherscan.io/address/0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c). This approach is applied to penalties that are used to address protocol operational costs (e.g., `removalCharge`).
+
+Penalized funds are burned for all the reasons described in the previous section. At the moment, only one penalty transferred to the Treasury is the `removalCharge`.
+
+## Bond shortage
+If, after penalties have been applied, a Node Operator's bond is less than required to cover the current Node Operator's validators, all new rewards will be used to replenish the NO bond until it is back to the required level. Node Operators can also "top-up" the bond themselves (by submitting the required difference) to be able to claim new rewards.
+
+If the amount of the penalty exceeds the amount of the Node Operator bond available, all available funds are burned.
```

Comment:
Yes, that's correct. During CSM design, we considered possible debt. We concluded that debt would most likely be useless in the permissionless module. Let's have a closer look at the examples.

There are 2 ways to lose the whole bond: extreme slashing or MEV stealing. We will focus on the second, since in case of slashing, validators are exited, and instead of repaying debt, it will be easier for the user to create a new operator simply. In case of significant MEV stealing resulting in a full bond loss, all active validators will be requested to exit as they are now unbonded. If the validators are exited, we are in a case similar to the first one. If not, they will become stuck. Hence, Performance Oracle will not allocate new rewards to the Node Operator. Yet again, quitting and creating a new Operator is easier.